### PR TITLE
fix type normalization

### PR DESCRIPTION
**Pull request by drewctate**
#354 and  #347

### CODE REVIEW

File: packages/content/src/portal.ts
Comment (tomwayson) [inline packages/content/src/portal.ts:112]: I think we still need to support string here, just pas it in as `{ type: itemOrType }` when it's a string and add this to  #305

**Review by tomwayson — CHANGES_REQUESTED** [review]
just need to un break the breaking changes, but otherwise this looks really good. Also, should wait to merge this until after #357 is merged and released.

File: packages/common/src/content.ts
Comment (drewctate) [reply packages/common/src/content.ts:62]: Fair point. I agree since this isn't a nested prop.

File: packages/common/src/content.ts
Comment (drewctate) [reply packages/common/src/content.ts:53]: Good call

File: packages/content/src/portal.ts
Comment (drewctate) [reply packages/content/src/portal.ts]: Done

**Review by tomwayson — APPROVED** [review]

**Pull request merged**
